prisma/schema.prisma

```diff
@@ -27,6 +27,8 @@ model Organization {
   id          String     @id @default(cuid())
   name        String
   address     String
+  email       String     @unique
+  password    String
```

Comment:
Why do youneed email and password on organization?

Reply:
Because I was of the opinion that an organization should be able to send invites to their employees and as such I felt that I'll need some form of authentication. Hence, why I added an email and password.